<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentSignUp</class>
 <widget class="QDialog" name="StudentSignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Sign Up</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,1,2">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Student sign up</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Subject 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="subject1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Subject 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="subject2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Subject 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="subject3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addStudent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add student</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
